### ISSUE TITLE
Lab Report 5 Feedback

### ISSUE THREAD

**Issue by emjinn** [andrewonozuka/cse15l-lab-reports#5]
Here is your feedback for Lab Report 5:
	
- [x] Provides correct description of how mismatch in outputs was identified
- [x] Correct markdown formatting.
- [x] Lab report is on its own page, not the index page.
 
Test File 1
- [x] Describes which implementation is correct
- [x] Shows actual output and the expected output 
- [ ] Provides correct description of the bug
- [ ] Provides code snippet that needs to be changed

Test File 2
- [x] Describes which implementation is correct
- [x] Shows actual output and the expected output 
- [ ] Provides correct description of the bug
- [ ] Provides code snippet that needs to be changed
 
 
Additional comments from your grader:
 
Please provide screenshots of the expected output as well! In addition, for each test file, you need to describe the bug and explain how you're able to address the issue in your code! In your code snippet screenshot, please highlight the area that you want to address.

Feel free to comment below with any questions about the rubric/feedback
 
--------
 
If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:
 
Link to this issue
Link to your published lab report page

**Comment by AndrewOnozuka** [comment]
I'm confused why I lost points for the "shows actual output" since you say in the comments "Please provide screenshots of the expected output as well!" does that not entail that the actual output screenshots are present? (which they are).

Also in the lab report write up it says: "For the implementation that’s not correct (or choose one if both are incorrect), describe the bug (the problem in the code) in about 2-3 sentences." So I was under the assumption that I only needed to explain one? Is the write up incorrect?

**Comment by emjinn** [comment]
Hi Andrew!

My apologies, I missed the screenshots when grading your report -- just adjusted the points accordingly! 

In regards to your second question, you are required to describe the bug for _each_ test file. From the screenshots that you had included in your lab report, it appears that "markdown-parser" provides the incorrect output for both test files. 

Thus, for both test files, you should describe in 2-3 sentences _each_ why markdown-parser is generating the incorrect output (i.e. actual output doesn't match expected output). Once you have identified the reason _why_ for both test files, try to find a snippet of code that you want want to focus on for each test file in order to address the bug, so the implementation is able to provide the correct output. 

**TLDR: You should have a description of the bug and a snippet of code for _each_ test file that you had analyzed. Total of two descriptions of the bug & two snippets of code in your lab report!** 

Please don't hesitate to reach out if you need clarification on anything I said above! 

Best,
Emily
